### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by mbychkowski**
Added some styles the app

### CODE REVIEW

File: mapStyles2.js [hunk 1]
```diff
@@ -0,0 +1,331 @@
+var mapStyles = [
```
Comment (mjsimon91) [inline mapStyles2.js:1]: I assumed this is a google thing, but what is the reason for two separate files? Won't one over write the other?

File: index.html [hunk 2]
```diff
@@ -114,6 +114,7 @@ <h5 class="disoveryName">Title</h5>
     <script type="text/javascript" src="./checkbox.js"></script>
     <script type="text/javascript" src="./venue.js"></script>
     <script type="text/javascript" src="./discovery.js"></script>
+    <script type="text/javascript" src="./mapStyles2.js"></script>
     <script type="text/javascript" src="./script.js"></script>
     <script type="text/javascript" src="./table.js"></script>
```
Comment (mjsimon91) [inline index.html:120]: It doesnt look like the mapstyles.js was added. Was this on purpose?

**Review by mjsimon91 — COMMENTED** [review]
There are 2 mapstyles files (mapstyles.js and mapstyles2.js but only 2 seems to be linked in the index.html file. Is this on purpose?

File: index.html (hunk 2 above)
Comment (mbychkowski) [reply index.html:120]: I thought I would keep 2 map styles for now. Each is different in how it makes the map look. By simply switching the link to 1 or 2 we will get a different style. Eventually we will just decide on 1, but i was a litt unsure which one looked better

File: mapStyles2.js (hunk 1 above)
Comment (mbychkowski) [reply mapStyles2.js:1]: Answered above. Not a google thing, but 2 different files for 2 different styles.

**Review by mjsimon91 — APPROVED** [review]

**Pull request closed without merging**